revive.toml

```diff
@@ -0,0 +1,52 @@
+ignoreGeneratedHeader = false
+severity = "error"
+confidence = 0.8
+errorCode = 0
+warningCode = 0
+
+[rule.blank-imports]
+[rule.context-as-argument]
+[rule.dot-imports]
+[rule.error-return]
+[rule.error-strings]
+[rule.error-naming]
+[rule.exported]
+[rule.if-return]
+[rule.var-naming]
+[rule.package-comments]
+[rule.range]
+[rule.receiver-naming]
+[rule.indent-error-flow]
+[rule.empty-block]
+[rule.superfluous-else]
+[rule.modifies-parameter]
```

Comment:
Maybe `increment-decrement` should be added?

Reply:
I don't see the point to `++` in go. Its something additional to learn even though it is no different than `+=1` which is already quite concise. I would